src/fuzzyduck.cpp

```diff
@@ -22,21 +22,32 @@ void FuzzyDuck::BeginFuzzing() {
 	if (max_queries == 0) {
 		throw BinderException("Provide a max_queries argument greater than 0");
 	}
+	auto &fs = FileSystem::GetFileSystem(context);
 	if (!complete_log.empty()) {
-		auto &fs = FileSystem::GetFileSystem(context);
 		TryRemoveFile(complete_log);
 		complete_log_handle =
 			fs.OpenFile(complete_log, FileFlags::FILE_FLAGS_WRITE | FileFlags::FILE_FLAGS_FILE_CREATE_NEW);
 	}
 	if (enable_verification) {
 		RunQuery("PRAGMA enable_verification");
 	}
+	if (!randoms_config_filepath.empty()) {
+		randoms_config_handle = fs.OpenFile(randoms_config_filepath, FileFlags::FILE_FLAGS_READ);
+		const char *file_path = "config.json";
+		auto config = GetConfigFromFile(file_path);
+		if (!config.size()) {
+			config = GetDefaultConfig();
+		}
+	}
 }
 
 void FuzzyDuck::EndFuzzing() {
 	if (complete_log_handle) {
 		complete_log_handle->Close();
 	}
+	if (randoms_config_handle) {
```

Comment:
you can probably close the file handle earlier, once you finish getting the config. No need to keep it open while you are fuzzing